tests/unit/test_metadata.py

```diff
@@ -0,0 +1,56 @@
+"""This is the testing area for app.py."""
+
+###########
+# IMPORTS #
+###########
+
+import hashlib
+
+import pytest
+from httpx import AsyncClient
+
+#########
+# TESTS #
+#########
+
+
+@pytest.mark.unit
+@pytest.mark.asyncio
+async def test_list_metadata(client: AsyncClient) -> None:
```

Comment:
This test is marked as unit but contains an external dependency on a dataset that isn't also created as a fixture. Do we have a mark to represent end to end tests or external integration tests?

Reply:
#8 The tests are still scuffed, they were not majorly changed in this refactor.